impl/graphsync_test.go

```diff
@@ -371,15 +381,45 @@ func TestGraphsyncRoundTrip(t *testing.T) {
 	assertComplete(ctx, t)
 
 	tracing := collectTracing(t)
-	require.ElementsMatch(t, []string{
-		"response(0)->executeTask(0)",
-		"request(0)->newRequest(0)",
-		"request(0)->executeTask(0)",
-		"request(0)->terminateRequest(0)",
-	}, tracing.TracesToStrings())
+
+	traceStrings := tracing.TracesToStrings()
+	require.Contains(t, traceStrings, "response(0)->executeTask(0)")
+	require.Contains(t, traceStrings, "request(0)->newRequest(0)")
+	require.Contains(t, traceStrings, "request(0)->executeTask(0)")
+	require.Contains(t, traceStrings, "request(0)->terminateRequest(0)")
+	require.Contains(t, traceStrings, "responseMessage(0)->loaderProcess(0)->cacheProcess(0)") // should have one of these per response
+	require.Contains(t, traceStrings, "request(0)->verifyBlock(0)")                            // should have one of these per block
+
 	processUpdateSpan := tracing.FindSpanByTraceString("response(0)")
 	require.Equal(t, int64(0), testutil.AttributeValueInTraceSpan(t, *processUpdateSpan, "priority").AsInt64())
 	require.Equal(t, []string{string(td.extensionName)}, testutil.AttributeValueInTraceSpan(t, *processUpdateSpan, "extensions").AsStringSlice())
+
+	// each verifyBlock span should link to a cacheProcess span that stored it
```

Comment:
this section verifies that the links properly exist between `verifyBlock` and `cacheProcess` spans